src/net.cpp

```diff
@@ -159,9 +159,9 @@ static std::vector<CAddress> convertSeed6(const std::vector<SeedSpec6> &vSeedsIn
 // Otherwise, return the unroutable 0.0.0.0 but filled in with
 // the normal parameters, since the IP may be changed to a useful
 // one by discovery.
-CAddress GetLocalAddress(const CNetAddr *paddrPeer, ServiceFlags nLocalServices)
+CAddress GetLocalAddress(const CNetAddr *paddrPeer, ServiceFlags nLocalServices, uint16_t default_listen_port)
```

Comment:
In commit "net: pass listen port into CConnman" (f2ad517e131142de1d28670d759c07bdf5466b3f)

Should be `port` or `listen_port`, not `default_listen_port` as far as I can tell. There are also some other uses of `default_listen_port` in this commit that look like they should be changed.